ch-05/src/Solution525.java

```diff
@@ -0,0 +1,16 @@
+public class Solution525 {
+    public static void main(String[] args) {
+        double piValue = 0;
+        int signChanger = 1;

```

Comment:
The scope of these variables is too large: you could declare them in the outer loop of line 6.